EXAMPLES.md

```diff
@@ -160,7 +154,8 @@ end
 ```
 
 In this case the `recursive: true` option mirrors the same options which are
-available to [`Net::{SCP,SFTP}`](http://net-ssh.github.io/net-scp/).
+available to [`Net::SCP`](http://net-ssh.github.io/net-scp/) and
```

Comment:
@JasonPoll can you change the SCP link to https://github.com/net-ssh/net-scp ? It looks like http://net-ssh.github.io/net-scp/  is out of date (the header says "Last Update: 2014").

Reply:
Can do.